### ISSUE TITLE
autofac exception when upgrading to v3

### ISSUE THREAD

**Issue by amoldeshpande** [Microsoft/BotBuilder#636]
trying to upgrade my bot to 1.3 and I get an 'Autofac.Core.DependencyResolutionException when I send a message from the emulator.

the inner exception is:

{"Value cannot be null.\r\nParameter name: ServiceUrl"}

This happening right in my Post() method where I do

          if (activity != null && activity.GetActivityType() == ActivityTypes.Message)
            {
                await Conversation.SendAsync(activity, () => new MyDialog());
            }

**Comment by amoldeshpande** [comment]
ok, I figured out that this was because I was till running the .2 version of the emulator (I guess I expected it to auto-update, but I see why it doesn't). Anyway, now I'm getting this (running a debug version of botbuilder that I built out of github)

iisexpress.exe Warning: 0 : Invalid token. System.ArgumentNullException: Value cannot be null.
Parameter name: securityToken
   at System.IdentityModel.Tokens.JwtSecurityTokenHandler.ValidateToken(String securityToken, TokenValidationParameters validationParameters, SecurityToken& validatedToken) in c:\workspace\WilsonForDotNet45Release\src\System.IdentityModel.Tokens.Jwt\JwtSecurityTokenHandler.cs:line 658
   at Microsoft.Bot.Connector.JwtTokenExtractor.<ValidateTokenAsync>d__9.MoveNext() in I:\github\BotBuilder\CSharp\Library\Microsoft.Bot.Connector\JwtTokenExtractor.cs:line 131
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.ConfiguredTaskAwaitable`1.ConfiguredTaskAwaiter.GetResult()
   at Microsoft.Bot.Connector.JwtTokenExtractor.<GetIdentityAsync>d__6.MoveNext() in I:\github\BotBuilder\CSharp\Library\Microsoft.Bot.Connector\JwtTokenExtractor.cs:line 73
Exception thrown: 'System.ArgumentNullException' in System.IdentityModel.Tokens.Jwt.dll
